backend/supervisor.d/services.conf

```diff
@@ -71,8 +71,8 @@ stdout_logfile_maxbytes=0
 
 [program:database]
 command=/usr/local/bin/python -c "import %(program_name)s; %(program_name)s.run()"
-process_name=%(program_name)s
-numprocs=1
+process_name=%(program_name)s_%(process_num)s
```

Comment:
In the wiki https://github.com/FORTH-ICS-INSPIRE/artemis/wiki#invoking-multiple-detectors-optional
we state:
```
process_name=%(program_name)s_%(process_num)02d
```
Shouldn't these 2 be consistent?

Reply:
Also note that since the configuration of supervisor (in local_configs) is decoupled from version control, we need to alert the users when some change like this happens

Reply:
I would recommend to document this in the wiki instead of changing the default configuration

Reply:
they are exactly the same